docs/developer-guide/godocs.md

```diff
@@ -1,3 +1,8 @@
 # Go docs
 
-Ligato Go Docs are available [here](https://godoc.org/github.com/ligato/vpp-agent)
+To access Ligato GoDocs, see the following:
+
+- [VPP agent Go docs](https://godoc.org/github.com/ligato/vpp-agent).
```

Comment:
Please update the link to https://pkg.go.dev/github.com/ligato/vpp-agent 
godoc.org has been replaced with pkg.go.dev

Reply:
done.